compiler_gym/envs/llvm/service/Cost.cc

```diff
@@ -236,6 +239,9 @@ LlvmCostFunction getCostFunction(LlvmRewardSpace space) {
       return LlvmCostFunction::TEXT_SIZE_BYTES;
 #endif
   }
+
+  LOG(FATAL) << "Unreachable";
```

Comment:
Consider `__builtin_unreachable();`

[clang lang extensions](https://clang.llvm.org/docs/LanguageExtensions.html)

Reply:
Oh cool, I hadn't seen that before, thanks :) If I understood the docs properly, that is only used as a compiler hint for the optimizer, so has undefined behavior if executed. In that case I think I want some combination of that intrinsic with the `LOG(FATAL)`, since that will trigger a runtime error if it is reached. Maybe something like:

```c++
#define UNREACHABLE(msg) DLOG(FATAL) << "Unreachable: " << (msg); __builtin_unreachable();
```